src/ontology/Makefile

```diff
@@ -17,11 +17,11 @@ catalog-v001.xml: default-catalog.xml
 
 # Note: we currently use no-reasoner for now, until we can trust all inferences
 build/hp-simple.obo: hp-edit.owl catalog-v001.xml
-	ontology-release-runner --catalog-xml catalog-v001.xml $(OORT_ARGS) --ignoreLock --skip-release-folder --outdir build --simple --allow-overwrite --no-reasoner $<
+	ontology-release-runner --catalog-xml catalog-v001.xml $(OORT_ARGS) --ignoreLock --no-subsets --skip-release-folder --outdir build --simple --skip-ontology-checks --allow-overwrite --no-reasoner $<
```

Comment:
@cmungall please comment if this change has consequences that we are maybe not aware of. thanks

Reply:
Should be fine. I think you should get rid of oort asap

Reply:
I have already replaced all this in another pull request, but we need more testing before we can merge that one. @drseb seems its fine.